ci-operator/step-registry/aws/provision/bastionhost/aws-provision-bastionhost-commands.sh

```diff
@@ -30,8 +30,8 @@ s3_bucket_name="${CLUSTER_NAME}-s3"
 bastion_ignition_file="${SHARED_DIR}/${CLUSTER_NAME}-bastion.ign"
 bastion_cf_tpl_file="${SHARED_DIR}/${CLUSTER_NAME}-bastion-cf-tpl.yaml"
 
-if [[ ! -f "${bastion_ignition_file}" ]]; then
-  echo "'${bastion_ignition_file}' not found, abort." && exit 1
+if [ "${BASTION_HOST_AMI}" == "" ] && [[ ! -f "${bastion_ignition_file}" ]]; then
```

Comment:
Shall the conditions be of OR relationship instead?

Reply:
Should be `AND`, the host can be provisioned if one of the following conditions is met:
1. user provides `BASTION_HOST_AMI` (not empty)
2. use dynamic host (ignition file exits)

So if both of them are not met, will abort the process.

Reply:
Got it, anyway it looks a bit strange to me, e.g. if BASTION_HOST_AMI is specified, the ignition file would be ignored.

Reply:
@yunjiang29 suggest using `[[ "${BASTION_HOST_AMI}" == "" ]]`

Reply:
all comments addressed, thank you